scripts/scripts.js

```diff
@@ -78,13 +85,82 @@ function buildHeroBlock(main) {
   }
 }
 
+/**
+ * Build Prev Next Bottom Navigation
+ * @param main
+ * @returns {Promise<void>}
+ */
+async function buildPrevNext(main) {
```

Comment:
you could reduce redundancy and increase readability by using functional programming, for example in the area of creating the arrow and insertion of nav/arrow elements for prev and next box, plus using a guard clause:

```
async function buildPrevNext(main) {
  const moveHeaderLinkDiv = (el) => {
    const text = el.innerText;
    const newDiv = document.createElement('div');
    newDiv.classList.add('prev-next-header-link');
    newDiv.innerText = text;
    el.innerText = '';
    el.insertAdjacentElement('afterbegin', newDiv);
  };

  const buildNavButtons = async (key, defaultText) => {
    const newDiv = document.createElement('div');
    newDiv.classList.add('prev-next-button');
    newDiv.innerText = await translate(key, defaultText);
    return newDiv;
  };

  const createArrow = (direction) => {
    const arrow = document.createElement('div');
    arrow.classList.add(`${direction}-arrow`);
    arrow.innerHTML = direction === 'left' ? HTML_ARROW_PREV : HTML_ARROW_NEXT;
    return arrow;
  };

  const prevNext = main.querySelector('.columns.prev-next');
  if (!prevNext) return;

  const prevNextContainer = prevNext.firstElementChild;
  const verticalSeparator = document.createElement('div');
  verticalSeparator.classList.add('vertical-line');
  prevNextContainer.firstElementChild.insertAdjacentElement('afterend', verticalSeparator);

  const prevBox = prevNextContainer.firstElementChild.querySelector('a');
  const nextBox = prevNextContainer.lastElementChild.querySelector('a');

  if (prevBox) {
    moveHeaderLinkDiv(prevBox);
    prevBox.insertAdjacentElement('afterbegin', await buildNavButtons('navPreviousText', 'Previous'));
    prevBox.insertAdjacentElement('afterbegin', createArrow('left'));
  }

  if (nextBox) {
    moveHeaderLinkDiv(nextBox);
    nextBox.insertAdjacentElement('afterbegin', await buildNavButtons('navNextText', 'Next'));
    nextBox.insertAdjacentElement('afterbegin', createArrow('right'));
  }
}

```